pages/org/quotient/index.jsx

```diff
@@ -145,78 +112,33 @@ const QuotientBankForm = () => {
     // eslint-disable-next-line
   }, [verified]);
 
-  useEffect(() => {
-    const [govId, webcamPic] = [form.getValues('govId'), form.getValues('webcamPic')];
-    if (isCaptureCompleted && webcamPic === '') {
-      form.resetField('webcamPic', { defaultValue: '/example_webcam.png' });
-    }
-    if (isUploadPoDComplete && govId === '') {
-      form.resetField('govId', { defaultValue: '/example_passport.png' });
-    }
-  }, [isCaptureCompleted, isUploadPoDComplete, form]);
-
-  async function onSubmit(values) {
-    console.log('values', values);
-    setIsLoading(true);
-    toast.success('Bank account created, please proceed to next step.');
-
-    setTimeout(() => {
-      console.log('setters true');
-      setIsSuccess(true);
-      setIsLoading(false);
-    }, 1000);
-  }
-
   return (
     <>
       <Head>
         <title>Quotient - Open New Bank Account</title>
       </Head>
       <Header />
       <LoadingModal isLoading={isLoading} setIsLoading={setIsLoading} />
-      {isSuccess ? (
-        <div className='mainContainer'>
-          <QuotientSuccess
-            title={'Your account has been opened!'}
-            proofTemplateId={proofTemplateId} />
-        </div>
-      ) : (
-        <div className='p-4 min-h-screen mainContainer'>
-          <div className='mb-4 mt-2'>
-            <h2 className='font-semibold text-2xl'>Open New Banking Account</h2>
-          </div>
-          <Form {...form}>
-            <form onSubmit={form.handleSubmit(onSubmit)}>
-              <div className='flex gap-4 flex-wrap'>
-                <div className='flex-1 w-full xl:w-2/3 md:w-2/3 p-4 bg-neutral-50 rounded-lg space-y-5'>
-                  <FormFieldNameAndBirthday control={form.control} dob={true} applicant={applicant} />
-                  <Separator />
-                  <FormFieldAddress control={form.control} applicant={applicant} />
-                  <Separator />
-                  <FormFieldPersonalContact control={form.control} isUsaCitizen={true} />
-                </div>
-                <div className='flex-2 w-full md:w-1/3 xl:w-1/3'>
-                  <FormFieldGovId
-                    control={form.control}
-                    isSelfieCaptureCompleted={isCaptureCompleted}
-                    setIsCaptureCompleted={setIsCaptureCompleted}
-                    isDocumentCaptureComplete={isUploadPoDComplete}
-                    setIsUploadPoDComplete={setIsUploadPoDComplete}
-                  />
+      <div className='mainContainer'>
+          {
+            showSuccess ? (
+              <div className='flex flex-col items-center justify-center min-h-[60vh]'>
+                <div className='bg-white rounded-2xl shadow-lg p-8 flex flex-col items-center max-w-xl w-full'>
+                  <CheckCircle2 className='w-16 h-16 text-green-500 mb-4' />
+                  <h2 className='text-2xl font-bold mb-2 text-center'>Verification Successful!</h2>
```

Comment:
I'd probably make the header more focused on "Bank Account Opened" - I don't think "Verification" will mean much to the viewers.